src/UriTemplates/UriTemplateTable.cs

```diff
@@ -14,11 +14,11 @@ public void Add(string key, UriTemplate template)
             _Templates.Add(key,template);
         }
 
-        public TemplateMatch Match(Uri url)
+        public TemplateMatch Match(Uri url, QueryStringParameterOrder order = QueryStringParameterOrder.Strict)
```

Comment:
This is going to be a binary breaking change.  We should keep the overload without the parameter.

Reply:
I'd say moving to only supporting netstandard2.0 is probably a breaking change anyway so....??

Reply:
Ok, fine, so then let's bump the major version number.